docs/v0.7/en/reference/sql/information-schema/table-constraints.md

```diff
@@ -0,0 +1,2 @@
+# TABLE_CONSTRAINTS
+todo
```

Comment:
we do not have a table named `TABLE_CONSTRAINTS`.

Reply:
Yep, so it's `TODO`, we have to support it.